pages/1.9/release-notes/1.9.7/index.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.9.7
+title: Release Notes for 1.9.7
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.9.7.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.7
+
+- CORE-1375 - Docker executor no longer hangs due to lost messages.
```

Comment:
nit: Although it seems that the standard is to end every item on a period, common practice is to only do this if the item is a complete sentence.

Reply:
Formatting is consistent with previous versions of release notes.